database/seeds/DonationsTableSeeder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class DonationsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+      //Seeding data on Donation table
+
+      \App\Donations::create(
+        array(
+          'donor_id' => '3',
+          'receiver_id' => '4'
+        )
+      );
+      \App\Donations::create(
+        array(
+          'donor_id' => '2',
+          'receiver_id' => '4'
+        )
+      );
+      \App\Donations::create(
```

Comment:
- use faker to seed data
- don't use Fully qualified class names. e.g. `\App\Donations`

```
use `App\Donations`;
```